measures/PowerOutage/measure.xml

```diff
@@ -1,9 +1,25 @@
 <measure>
   <schema_version>3.0</schema_version>
+  <error>C:/OpenStudio-BuildStock-WH-GEB-ToMerge/resources/measures/HPXMLtoOpenStudio/resources/util.rb:1160: syntax error, unexpected tCONSTANT, expecting keyword_then or ';' or '\n'
+...| water_heater.is_a? OpenStudio::Model::WaterHeaterStratifie...
+...                               ^
+C:/OpenStudio-BuildStock-WH-GEB-ToMerge/resources/measures/HPXMLtoOpenStudio/resources/util.rb:1166: syntax error, unexpected keyword_elsif, expecting keyword_end
+      elsif water_heater.is_a? OpenStudi...
+           ^
+C:/OpenStudio-BuildStock-WH-GEB-ToMerge/resources/measures/HPXMLtoOpenStudio/resources/util.rb:1309: syntax error, unexpected tCONSTANT, expecting keyword_end
+...| water_heater.is_a? OpenStudio::Model::WaterHeaterStratifie...
+...                               ^
+C:/OpenStudio-BuildStock-WH-GEB-ToMerge/resources/measures/HPXMLtoOpenStudio/resources/util.rb:1410: syntax error, unexpected tCONSTANT, expecting keyword_end
+...| water_heater.is_a? OpenStudio::Model::WaterHeaterStratifie...
+...                               ^
+C:/OpenStudio-BuildStock-WH-GEB-ToMerge/resources/measures/HPXMLtoOpenStudio/resources/util.rb:1531: syntax error, unexpected tCONSTANT, expecting keyword_end
+...| water_heater.is_a? OpenStudio::Model::WaterHeaterStratifie...
+...                               ^
+C:/OpenStudio-BuildStock-WH-GEB-ToMerge/resources/measures/HPXMLtoOpenStudio/resources/util.rb:1921: syntax error, unexpected keyword_end, expecting end-of-input</error>
```

Comment:
👆 Looks like something needs to be fixed in resources/measures/HPXMLtoOpenStudio/resources/util.rb.

Reply:
This should fix it: https://github.com/NREL/resstock/pull/483/commits/c2b4083b4f533ebdb453af3d60f25ba6e0b2f596